projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -409,30 +404,6 @@ export class IgxSliderComponent implements
         }
     }
 
-    /**
-     * Returns if the {@link IgxSliderComponent} is set as continuous.
-     * ```typescript
-     * @ViewChild("slider2")
-     * public slider: IgxSliderComponent;
-     * ngAfterViewInit(){
-     *     let continuous = this.slider.continuous;
-     * }
-     * ```
-     */
-    @Input()
-    @DeprecateProperty(`IgxSliderComponent \`isContinuous\` property is deprecated.\nUse \`continuous\` instead.`)
```

Comment:
I can't find which version we've deprecated this in?

Reply:
As I see from the changelog [here](https://github.com/IgniteUI/igniteui-angular/pull/4994/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR65) this change was made in [7.3.4 ](https://github.com/IgniteUI/igniteui-angular/releases/tag/7.3.4) release, even though the label that was set to the related [issues](https://github.com/IgniteUI/igniteui-angular/pull/4994) is 8.0.x. I've gone through all releases we've made so far and in [7.3.4](https://github.com/IgniteUI/igniteui-angular/releases/tag/7.3.4) is the last place where we are setting **isContinuous** as deprecated.